server/migrations/helpers.js

```diff
@@ -0,0 +1,50 @@
+function calculatePreferences(user) {
+  const updatedPreferences = {
+    config: '',
+    sort: 0,
```

Comment:
I believe you can remove the setting of these attributes at the end if you initialize the new object with them.

```suggestion
    config: user.preferences?.config || '',
    sort: user.preferences?.sort || '',
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/241